src/main/kotlin/hu/kirdev/schpincer/web/ConfigureController.kt

```diff
@@ -408,6 +408,8 @@ class ConfigureController {
     @Throws(FileNotFoundException::class, DocumentException::class)
     fun showOpenings(openingId: Long,
                      type: String?,
+                     page: String?,
+                     emptyrows: Long = 0,
```

Comment:
Does this need to bee of type Long? A basic Int should be enough

Reply:
Just applied all your suggestions in the next commit, thanks!